ui/widgets/button.js

```diff
@@ -12,20 +12,22 @@
 //>>description: Enhances a form with themeable buttons.
 //>>docs: http://api.jqueryui.com/button/
 //>>demos: http://jqueryui.com/button/
-//>>css.structure: ../themes/base/core.css
-//>>css.structure: ../themes/base/button.css
+//>>css.structure: ../themes/base/core.css, ../themes/base/button.css
```

Comment:
Why did you change this? We have multiple lines for these in every other widget. Though I guess we could change them all: https://github.com/rxaviers/builder-jquery-css/blob/master/index.js#L8